libcxx/utils/generate_escaped_output_table.py

```diff
@@ -328,8 +313,13 @@ def generate_data_tables() -> str:
     #
     # When this region becomes substantially smaller we need to investigate
     # this design.
+    #
+    # Due to P2713R1 Escaping improvements in std::format the range
+    #   E0100..E01EF  ; Grapheme_Extend # Mn [240] VARIATION SELECTOR-17..VARIATION SELECTOR-256
+    # is no longer part of these entries. This causes an increase in the size
+    # of the table.
     assert data[-1].upper == 0x10FFFF
-    assert data[-1].upper - data[-1].lower > 900000
+    # assert data[-1].upper - data[-1].lower > 900000
```

Comment:
Is this intended to be commented out? Why?

Reply:
It has been removed the next patch does something here again.